gitregostore/gitstoremethods.go

```diff
@@ -249,7 +249,7 @@ func (gs *GitRegoStore) GetOPAFrameworkByName(frameworkName string) (*opapolicy.
 	for _, fw := range gs.Frameworks {
 		if strings.EqualFold(fw.Name, frameworkName) ||
 			// If backward compatibility is supported,try to compare the new CIS name.
-			(supportBackwardCompatibility && strings.EqualFold(fw.Name, newFrameworkName(frameworkName))) {
+			(true && strings.EqualFold(fw.Name, newFrameworkName(frameworkName))) {
```

Comment:
@YiscahLevySilas1 @kooomix ?

Reply:
We needed to remove it because a control was added with an idea that broke the future control ids sequence.

We will restore it back soon.